src/Gui/QListWidgetDragBugFix.cpp

```diff
@@ -0,0 +1,58 @@
+/***************************************************************************
+ *   Copyright (c) 2023 Boyer Pierre-louis <pierrelouis.boyer@gmail.com>   *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "PreCompiled.h"
+#ifndef _PreComp_
+# include <QDragMoveEvent>
+#endif
+
+#include "QListWidgetDragBugFix.h"
```

Comment:
Name is really not very good. :)
First you should never start with `Q` as this denotes a class genuinely provided by Qt.
So something like `ListWidgetDragDrop` ?

Reply:
You are corerct I shouldn't have started it with Q. I dumbly reused the previously used 'QListWidgetCustom' and replaced Custom with a better name.
Regarding the name, I think it's a good name because it reminds us the exact purpose of this class, that is to fix the qt bug. And that later when Qt fix the bug this will not be necessary anymore.
I think `ListWidgetDragDrop` might be misleading in the way that it does not clearly state that this should be temporary until qt fixes the bug.

Reply:
Does the FreeCAD project have a published standard for variable naming in C++ code?  I know there's some general conventions and I've seen some forum posts about it but can't find an official standard.

Reply:
> Does the FreeCAD project have a published standard for variable naming in C++ code? 

Don't think so.